docs/attributes-registry/vcs.md

```diff
@@ -0,0 +1,37 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# VCS
+
+## VCS Repository Attributes
+
+This group defines the attributes for [Version Control Systems (VCS)](https://en.wikipedia.org/wiki/Version_control).
+
+| Attribute                     | Type   | Description                                                                                                                                                                                | Examples                                                                                                                             | Stability                                                        |
+| ----------------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------- |
+| `vcs.repository.change.id`    | string | The ID of the change (pull request/merge request) if applicable. This is usually a unique (within repository) identifier generated by the VCS system.                                      | `123`                                                                                                                                | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `vcs.repository.change.title` | string | The human readable title of the change (pull request/merge request). This title is often a brief summary of the change and may get merged in to a ref as the commit summary.               | `Fixes broken thing`; `feat: add my new feature`; `[chore] update dependency`                                                        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `vcs.repository.ref.name`     | string | The name of the [reference](https://git-scm.com/docs/gitglossary#def_ref) such as **branch** or **tag** in the repository.                                                                 | `my-feature-branch`; `tag-1-test`                                                                                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `vcs.repository.ref.revision` | string | The revision, literally [revised version](https://www.merriam-webster.com/dictionary/revision), The revision most often refers to a commit object in Git, or a revision number in SVN. [1] | `9d59409acf479dfa0df1aa568182e43e43df8bbe28d60fcf2bc52e30068802cc`; `main`; `123`; `HEAD`                                            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Are `main` and `HEAD` valid revisions or would those rather be the `ref.name`?

Reply:
Yea. The footnote mentions that It can be identical to the `ref.name` and if it is the same, both values should be set.

Reply:
All good! 😊